lib/ledger_sync/adaptors/quickbooks_online/customer/operations/create.rb

```diff
@@ -27,12 +27,12 @@ def operate
 
             def local_resource_data
               {
-                'DisplayName': resource.name,
-                "PrimaryPhone": {
-                  "FreeFormNumber": resource.phone_number
+                'DisplayName' => resource.name,
+                'PrimaryPhone' => {
+                  'FreeFormNumber' => resource.phone_number
```

Comment:
Is phone number required to make a customer? I feel like this shoudln't be

Reply:
No its not. In schema validation we have it as `maybe`. But I would touch this code later. The issue is that right now we don't know if we just don't have phone number or we wanna remove it.

Either way, sending `'PrimaryPhone' => {'FreeFormNumber' => ''}` has same effect in QBO as not sending anything. It removes the phone number from user.

Reply:
@mattgmarcus Does this answer your question?  If so, please resolve.

Reply:
yes